spring-integration-hazelcast/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+
+group = 'org.springframework.integration'
+version = '0.0.1-SNAPSHOT'
```

Comment:
`version = '1.0.0-BUILD-SNAPSHOT'` and in the separate `gradle.properties` file.
It allows `Build-Master` to change the value on release automatically.